docker/oshmem_fedora/Dockerfile

```
@@ -29,5 +30,5 @@ ENV PATH=/home/shmem/openmpi-4.1.6/install/bin:"${PATH}" \
     OMPI_ALLOW_RUN_AS_ROOT=1 \
     OMPI_ALLOW_RUN_AS_ROOT_CONFIRM=1
 
-RUN ln -s /usr/bin/python3 /usr/bin/python
-RUN python -m pip install setuptools cffi numpy
+RUN python3 -m venv .venv
```

Just for the sake of clarity, shouldn't we be explicit about the full path to the virtual env?

```suggestion
RUN python3 -m venv /.venv
```

---

Same thing elsewhere below.

I'm also wondering whether we should just use `/venv`. The convention of a  hidden `.venv` is for when you create a scratch venv within the source tree of a project, but for a "global" usage like there, I'm not so sure.

---

@mrogowski Don't take my comments above as hard requests for changes.  I'm just considering alternatives.

---

Sounded like a good alternative - changed all to `/venv` 👍